### ISSUE TITLE
Only the first lint error is highlighted

### ISSUE THREAD

**Issue by gerardroche** [jhoff/SublimeLinter-contrib-php-cs-fixer#3]
A file with multiple lint errors will only have the very first lint error highlighted.

When you fix the first lint error, the next one will be highlighted.

Ideally all the lint errors could be shown.

I haven't looked into it, but it may be possible to use one of the alternate formats to get all the information about the current file linting errors e.g. the `junit` or `checkstyle` formats. See https://github.com/FriendsOfPhp/PHP-CS-Fixer#usage.

**Comment by jhoff** [comment]
I totally agree.

The primary problem is not the format of the output, but the diff format. Currently there are only two options, `sbd` and `udiff`, both of which need to be parsed in order to extract the proper lines to highlight. We're using `udiff` because of the two it's the only one that shows a line number that can be parsed out, but unfortunately, changes that are near each other will be grouped under one line number.

Long story short, the output of php-cs-fixer isn't great for this use case and just highlighting the first is the easiest without writing a full udiff output parser.

I'm going to keep this issue open for tracking though, in case someone wants to work on it.

**Comment by mercurykd** [comment]
I implemented highlighting on all lines where there will be changes. and also output a short diff to the tooltip

![image](https://user-images.githubusercontent.com/30900414/214381609-76b3b49d-ae5e-478f-bbee-24b08dc61e3f.png)
